sys-block/gpart/gpart-0.3~git.recipe

```diff
@@ -0,0 +1,49 @@
+SUMMARY="A tool for recovering MBR partition tables in hard disks"
+DESCRIPTION="Gpart is a small tool which tries to guess what partitions are \
+on a PC type, MBR-partitioned hard disk in case the primary partition table \
+was damaged."
+HOMEPAGE="https://github.com/baruch/gpart"
+COPYRIGHT="1999-2017 Michail Brzitwa"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="23b651d1e9c6ae1454e5d3fe315d7d4e145661c0"
+SOURCE_URI="https://github.com/baruch/gpart/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="c492b7dc677e4c6af197f9598ba815a4e14a03578ee8a963430529767bc6c947"
+SOURCE_FILENAME="gpart-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="gpart-$srcGitRev"
+#PATCHES="gpart-$portVersion.patchset"
```

Comment:
If the patch is required then you should uncomment this line.
If it is not required then this line should be removed as well as the patch itself.

Please ignore this comment in case I am missing something.

Reply:
Sorry,I forgot for uncomment patches